example.domain

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "Project Description",
+
+    "domain": ".is-local.org",
+    "subdomain": "palwalgang",
+
+    "owner": {
+        "email": "theelf.gg@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "7bbfe172-5d99-490e-925f-81d164429edc.id.repl.co",
+        "TXT": ["replit-verify=7bbfe172-5d99-490e-925f-81d164429edc"]
```

Comment:
TXT records are not supported alongside CNAME records. If you want to have CNAME, it must be the only record.